### DISCUSSION TITLE
Non-alphanumeric character issues

### DISCUSSION THREAD

**Discussion by temporarium** [pawelsalawa/sqlitestudio#3890] [Q&A] [answered]
Is it just me or are others having issues with **non-alphanumeric character entry** in the application?

I'm referring to **not being able to type a quote or single-quote character (" or '), nor any accented characters (such as é, í, etc.)**.

I'm on version 3.2.1, on a Debian-based distro. I can type these characters in every other application. In fact, I'm quite often forced to type the quoted text (for queries, for example) into a note app and then copy/paste into SQLiteStudio.

Any ideas/suggestions?

**Comment by temporarium** [comment]
Very quiet here :-)

No one can confirm or deny if they have this issue?

**Reply by pawelsalawa** [reply]
It seems somewhat related to #3382 and #3069, although I'm not sure if it's exactly same issue. Also I have a problem to reproduce such issues. Unfortunately it has always worked for me (under all of Ubuntu, Windows, MacOS) with Polish characters.

Are you able to provide 100% reproducible recipe? Perhaps a VirtualBox image that reproduces this?

**Comment by temporarium** [comment, marked as answer]
![2020-12-27_08-59](https://user-images.githubusercontent.com/4754345/103172512-15659980-4822-11eb-9f75-cdd6197d0d25.png)

Thank you for your reply, @pawelsalawa !

I had tried different keyboard layout settings previously, but couldn't find the problem. Your reply made me try again...

For years, my favorite layout was International with dead keys, which I use for Spanish. _That_ was the issue. Switching to **AltGr dead keys** seems to have resolved my issue with the quote (" ') characters, as well as not being able to enter any accented chars (with diacritics, like é á í).
